modules/contracts/apps/challenge/Cargo.toml

```diff
@@ -41,6 +41,8 @@ abstract-core = { workspace = true }
 abstract-app = { workspace = true }
 abstract-sdk = { workspace = true }
 
+chrono = { workspace = true }
```

Comment:
Can you compile and check if wasm deployable on cosmos. Might be wrong, but I remember I've had issues with this crate after `wasmbind` feature got enabled by default. Cosmwasm has limited list of accepted methods